zavod/docs/best_practices/patterns.md

```diff
@@ -33,6 +33,111 @@ entity.add("name", row.pop("full_name"))
 entity.add("birthPlace", row.pop("place_of_birth"))
 ```
 
+The method `entity.add` works seamlessly with both a single string and a list of strings. This means that you don’t have to loop through a list to add multiple items individually. Instead, you can simply add the entire list at once.
```

Comment:
In the long run, we probably want to make the typing of `entity.add` more strict and take only one argument ... perhaps we could already say that it's not worth forcing values into a list, and nicer to put them in individually if they come that way :)

Reply:
@pudo do you mean we should prefer

```python
for name in h.multi_split(name, SPLITS):
    entity.add(name)
```

or just to note inn these docs that lists are accepted but don't do `entity.add([name])`?

or something else?